### ISSUE TITLE
Is there any plan to merge the protocol extension to upstream?

### ISSUE THREAD

**Issue by angl** [Microsoft/GVFS#22]
The protocol seems to be a useful extension to provide finer-grained access to the git storage anyway. Would love to see it's available in the standard git distribution!

**Comment by sanoursa** [comment]
Yes we certainly hope to do so. The way we're thinking about the problem is that there are features we want to push into upstream git because they would be useful for any virtualization implementation. Allowing for single-blob downloads is an important part of any such solution.

**Comment by angl** [comment]
Yeah, that would be very useful, and IMHO is the right way to go to scale git clients. I'm curious about whether you have presented the idea to the git maintainers, and if so what was their response?

**Comment by sanoursa** [comment]
We are still in the process of creating a patch series and proposal for these protocol extensions. I'll link it here once we have it.

**Comment by angl** [comment]
Thanks a lot! It would be quite useful for us as we are also looking at
something similar..

**Comment by bradenhs** [comment]
Any updates on this?
